eras/conway/impl/src/Cardano/Ledger/Conway/PParams.hs

```diff
@@ -1179,7 +1215,17 @@ upgradeConwayPParams UpgradeConwayPParams {..} BabbagePParams {..} =
     , cppProtocolVersion = toNoUpdate @f @ProtVer bppProtocolVersion
     , cppMinPoolCost = THKD bppMinPoolCost
     , cppCoinsPerUTxOByte = THKD bppCoinsPerUTxOByte
-    , cppCostModels = THKD bppCostModels
+    , cppCostModels =
```

Comment:
Could you please add a property test for Conway pparams upgrade that ensures that all fields have been upgraded along the lines of:

```haskell
propConwayPParamsUpgrade :: UpgradeConwayPParams Identity -> PParams Babbage -> Property
propConwayPParamsUpgrade ppu pp = property $ do
  let pp' = upgradePParams ppu pp :: PParams Conway
  pp' ^. ppPoolVotingThresholdsL `shouldBe` ucppPoolVotingThresholds ppu
  ...
  Map.lookup PlutusV3 (costModelsValid (pp' ^. ppCostModels)) `shouldBe` ucppPlutusV3CostModel ppu
```